syntax/vscode/README.md

```diff
@@ -0,0 +1,32 @@
+# logica-syntax-highlighting README
+
+## Quick Running
+
+In VS Code open this folder and press `F5` to open a new window with the extension loaded.
+
+## Development
+
+Install `js-yaml` as a development only dependency in the extension:
+
+```
+npm install js-yaml --save-dev
+```
+
+Edit `src/logica.tmLanguage.yaml`, which is written in TextMate grammar.
+
+Convert `yaml` to `json` with the command-line tool:
+
+```
+npx js-yaml src/logica.tmLanguage.yaml > syntaxes/logica.tmLanguage.json
+```
+
+
+## Installation
+
+To start using your extension with Visual Studio Code copy it into the `<user home>/.vscode/extensions` folder and restart Code.
```

Comment:
For users that didn't install VSCode extensions before (like myself :-) ), can you please give a command to do this?

I understand that "this extension" should be copied to the folder that you provided, but I don't know what should I copy the whole repository or yaml or json. Etc.

Having a command will save each user some time. :-)

Reply:
Apart from adding the `cp` command I'm also thinking about packaging and publishing the extension so that the user can install the extension more easily. The steps are described [here](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#publishing-extensions), where we need a Azure DevOps organization. I was wondering if I should create the organization myself or you can help me with this.